### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by fraenkel**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ COPY deployments/crds ./deployments/crds
 ENV CGO_ENABLED=0 GOOS=linux
 RUN go build -o hephaestus-controller ./cmd/controller
 
-FROM gcr.io/distroless/static:nonroot
+FROM gcr.io/distroless/static:debug-nonroot
```
Comment (ddl-dclegg) [inline Dockerfile:12]: Why `debug` and not just `non-root`?

**Review by ddl-dclegg — COMMENTED** [review]
Things look good generally, I just have the one question on the static image choice.

File: Dockerfile (hunk 1 above)
Comment (fraenkel) [reply Dockerfile:12]: that will be reverted... when debugging... I needed access to a shell to see what is going on.

Comment (fraenkel) [reply Dockerfile]: reverted

**Review by ddl-dclegg — APPROVED** [review]

**Pull request closed without merging**